rfc/2023/2023-05-01-va-rfc-frontend-performance.md

```diff
@@ -0,0 +1,156 @@
+# RFC: VFS teams shall monitor performance and gradually adopt performance budgets
+
+<!--
+The title is what you want comments on. Use the active voice in a future tense.
+Example:
+    - The website will be built using the XZY framework
+-->
+
+* Comment Deadline: `2023-06-10`
+* Team Crew and Name: <!--e.g. Platform/AMT-->
+* Authors:
+  * [@dcloud](https://github.com/dcloud)
+  * [Contributing Author 1 Github Handle](https://github.com/ContributingAuthor1Handle)
+  * [Contributing Author 2 Github Handle](https://github.com/ContributingAuthor2Handle)
+* [Original RFC Pull Request](https://github.com/department-of-veterans-affairs/va.gov-platform-arch/pull/-1)
+
+## Background
+
+> Explain the current state. What is the problem? What needs to happen? Provide enough background for someone new to the problem space to understand this decision. Use active voice, present tense, and decisive language.
+> Example: We need to choose how the website will render in the browser. There are several competing frameworks to choose from.
+
+Currently, VA.gov frontend pages perform poorly for a significant number of Veterans. In late 2022, 48.35% of users to va.gov were on relatively slow mobile devices (Source: Google Analytics, 7/26/22-8/01/22). A significant amount are also on slow third-party devices, such as library desktops, per [government research](https://docs.fcc.gov/public/attachments/DOC-357270A1.pdf). Automated reports on the [Frontend support dashboard](https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/frontend-support-dashboard/lighthouse-performance-report/) and in Datadog via [RUM's Core Web Vitals](https://docs.datadoghq.com/real_user_monitoring/browser/monitoring_page_performance/#core-web-vitals) show us that many pages on VA.gov perform poorly for Veterans. We need to optimize VA.gov frontend applications for better performance.
```

Comment:
> Currently, VA.gov frontend pages perform poorly for a significant number of Veterans.

I think it's safe to say the page perform poorly for everyone given the extremely low lighthouse scores. Maybe it can be called out that pages are under performing compared to general industry standard and this is especially prevalent on (list of situations)